docs/concepts/native-interop.rst

```diff
@@ -0,0 +1,260 @@
+Native interoperability
+=======================
+
+In this document we will dive a little bit deeper into all three ways of doing 
+"native interoperability" that are available on the .NET platform. 
+
+To refresh your memory, the two main ways of doing native interoperability are:
+
+* Platform Invoke (P/Invoke)
+* COM interoperability
```

Comment:
OK, then I will change it to be clearer. Still think it should be there if the Primer is general purpose.